<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwPlaneParametersWidget</class>
 <widget class="QWidget" name="igwPlaneParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>平面参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showPlaneCheckBox">
       <property name="text">
        <string>显示平面</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Origin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OriginXEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="OriginYEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="OriginZEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Normal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NormalXEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NormalYEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NormalZEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="XNormalButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X Normal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="YNormalButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Y Normal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ZNormalButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Z Normal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
